### ISSUE TITLE
Option to disable checking users through a trust?

### ISSUE THREAD

**Issue by joejsullivan** [Dionach/NtdsAudit#22]
Love the functionality, sometimes. For me running this against 9 DIT's where it pulls details from through the trust takes many, many, hours to complete. I don't need or want the info via the trusts. 
Any option or way to omit checking users via the trust?

Joe

**Comment by ph1ll** [comment]
It would be using an offline copy of NTDS.dit, so no details should be "pulled" via a trust and slow things down. If it is taking a very long time to complete, I expect that will just be down to the size of the NTDS file. Is there a particular step which is running very slow? `--debug` may help to identify.

There is an option to filter by OU, but as its still needs to parse much of the database to determine if an entry is relevant, I think any speed gains would be marginable.

**Comment by joejsullivan** [comment]
> It would be using an offline copy of NTDS.dit, so no details should be "pulled" via a trust and slow things down. If it is taking a very long time to complete, I expect that will just be down to the size of the NTDS file. Is there a particular step which is running very slow? `--debug` may help to identify.
> 
> There is an option to filter by OU, but as its still needs to parse much of the database to determine if an entry is relevant, I think any speed gains would be marginable.

Thanks Ph1ll
Let me rephrase that. It's obtaining users via offline copy of the DIT, due to the DC's also being GC's, I am getting all user accounts. I suppose my only option at that point would be to filter by OU.

I'll run one through a debug, right now, at around 96%, it hangs for long periods. 
Joe
